pkg/ansible/runner/runner_test.go

```diff
@@ -23,6 +26,33 @@ import (
 )
 
 func TestNewFromWatches(t *testing.T) {
+	cwd, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("unable to get working director: %v", err)
+	}
+
+	validTemplate := struct {
+		ValidPlaybook string
+		ValidRole     string
+	}{
+
+		ValidPlaybook: filepath.Join(cwd, "testdata", "playbook.yml"),
+		ValidRole:     filepath.Join(cwd, "testdata", "roles", "role"),
+	}
+
+	tmpl, err := template.ParseFiles("testdata/valid.yaml.tmpl")
+	if err != nil {
```

Comment:
No need to check the return error?
If not then just ignore the error:
```Go
tmpl, _ := template.ParseFiles("testdata/valid.yaml.tmpl")
```

Reply:
I think it makes sense to fail the test if we can't get the template, thoughts?

Reply:
I thought that was intentional at first but yes we should just fail if we can't parse the template.